### DISCUSSION TITLE
24gb GPU enough / supported?

### DISCUSSION THREAD

**Discussion by rav-en** [poloclub/diffusiondb#9] [Q&A]
Hi all, just curious on if a 24gb card is enough to get this working.
Also, is the quality of images any better than what Stability-AI/StableDiffusion or CompVis/Stable-Diffusion has achieved? 

Kind regards.

**Comment by xiaohk** [comment]
Hey @rav-en, thanks for reaching out!

There might be some confusion. DiffusionDB is not an ML model; it is a dataset of images generated by Stable Diffusion. Downloading this dataset and running some simple analysis do not require a GPU. Let me know if you have more question!
